### DISCUSSION TITLE
Will it be possible to combine Slidev with WYSIWYG editors?

### DISCUSSION THREAD

**Discussion by elliot-zzh** [slidevjs/slidev#936] [Ideas]
The default markdown editing system in VS Code does not support WYSIWYG. And so the CodeMirror editor embedded in Sildev. However, this feature, which is supported by editors such as Typora, Marktext, and Milkdown, is fantastic to me.

So is there a possibility of combining Slidev with WYSIWTG editors, such as Milkdown? Apart from the basic features of that kind of markdown editors, features such as page separating and animation preview might also be supported. This combination might come out as a VS Code extension, or a desktop app based on Tauri or Electron. (If it's a desktop app, what about naming it: **Slidesk** ?)

**Comment by george-gca** [comment]
Currently there is a [VSCode extension](https://github.com/slidevjs/slidev-vscode) for it.
